lib/ansible/plugins/action/reboot.py

```diff
@@ -50,7 +50,7 @@ class ActionModule(ActionBase):
     DEPRECATED_ARGS = {}  # type: dict[str, str]
 
     BOOT_TIME_COMMANDS = {
-        'freebsd': '/sbin/sysctl kern.boottime',
+        'freebsd': '/sbin/sysctl -x kern.boot_id',
```

Comment:
Please create a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment). See [this fragment](https://github.com/ansible/ansible/blob/c2f528b7686c9573aa8493357219e0bbe4416dd3/changelogs/fragments/reboot-change-default-boot-command.yaml) as an example.

Reply:
The change note has been included so I'm marking this thread as resolved.